### ISSUE TITLE
TypeScript Language Service, getApplicableRefactors() won't be called if user is not selecting something

### ISSUE THREAD

**Issue by cancerberoSgx** [TypeStrong/atom-typescript#1427]
TypeScript: Refactor selection only shows refactors when user select something. But since the signature of of that method is :

```typescript
function getApplicableRefactors(fileName: string, positionOrRange: number | ts.TextRange): ts_module.ApplicableRefactorInfo[] {
```
It's clear that the intention of the API is to show suggestions both when text is selected or not (notice `positionOrRange: number | ts.TextRange`

This is the same issue I reported to Visual Studio Code : https://github.com/Microsoft/vscode/issues/50392#issuecomment-391849789

They told me that that was the intended behavior, but now they changed it in the latest version. I strongly recommend you consider doing the same. 

[Example of a plugin](https://github.com/cancerberoSgx/typescript-plugins-of-mine/tree/master/typescript-plugin-proactive-code-fixes#declare-interfaces-from-return-values) - - in vscode I no longer need to select text to see refactors while in atom I still do

**Comment by lierdakil** [comment]
I believe this is resolved in v12.5.2. If you could test and get back with the results that would be nice.

**Comment by cancerberoSgx** [comment]
you are right.  I updated it early today so I thought I had the latest version. Sorry for the trouble. Thank you very much! keep it up!
